test/scripts/gh-actions/setup-deps.sh

```
@@ -105,6 +106,19 @@ else
   kubectl scale deployment lws-controller-manager -n lws-system --replicas=1
   kubectl patch deployment lws-controller-manager -n lws-system --type=json -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/resources", "value": {"requests": {"cpu": "20m", "memory": "64Mi"}, "limits": {"cpu": "100m", "memory": "256Mi"}}}]'
   kubectl wait deployment lws-controller-manager -n lws-system --for condition=Available --timeout=300s
+
+  if [[ $LLMISVC_AUTOSCALER == "hpa" ]]; then
+    echo "Installing LLMISVC HPA autoscaling components (Prometheus, Prometheus Adapter, WVA)..."
+    ${REPO_ROOT}/hack/setup/infra/manage.prometheus-helm.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)